src/Data/Container/Indexed/GreatestFixpoint.agda

```diff
@@ -0,0 +1,25 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Greatest fixpoint for containers
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K --guardedness #-}
+
+module Data.Container.Indexed.GreatestFixpoint where
+
+open import Level using (Level; _⊔_)
+open import Codata.Musical.M.Indexed
+open import Data.Container.Indexed using (Container)
+open import Relation.Unary using (Pred)
+
+private
+  variable
+    o c r : Level
+    O : Set o
+
+-- This lives in its own module due to its unsafe use of sized types.
```

Comment:
Pretty sure this is my fault, but this comment is wrong! Doesn't use sized types...